media_support/service.py

```
@@ -521,9 +574,15 @@ def _filter_video_duration_candidates(
         skipped_durations: list[float] = []
         for item in candidates:
             duration = item.duration_seconds
-            if duration is None:
-                duration = self._probe_remote_video_duration(item.url)
-                item.duration_seconds = duration
+            # 缺时长或大小时做一次 Range 探测，两个值一起拿。
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)